src/sentry/api/endpoints/organization_events.py

```diff
@@ -250,16 +250,7 @@ def get(self, request: Request, organization) -> Response:
             or batch_features.get("organizations:dashboards-mep", False)
         )
 
-        use_profiles = batch_features.get("organizations:profiling", False)
-
-        use_occurrences = options.get("performance.issues.create_issues_through_platform", False)
-
-        use_metrics_layer = batch_features.get("organizations:use-metrics-layer", False)
-
-        starfish_view = batch_features.get("organizations:starfish-view", False)
-
```

Comment:
I think `starfish` and `use_profiles` might still be needed here

Reply:
I think the think I'm confused about is that as soon as we enabled the option, then this would have been true for all requests to this api. So now that I'm removing it, then shouldn't it just always be `True`? Or was that a bug in how we handled the options to begin with?

Reply:
yeah the way we handled this here isn't 100% accurate. it's more that if you have any of the feature flags set, the dataset in the query params will be respected. but if you have starfish view enabled and pass in profiling dataset it would still work even though you don't have the profiling feature flag

Reply:
yeah `use_custom_dataset` isn't needed since it's going to be true most of the time anyway

Reply:
Right... I think it's ok, if they don't have the profiling feature flag they wouldn't have profiles anyway. If we need validation here it probably needs to associate feature flags with a specific dataset or something. I'll leave that for someone else to work on if we decide we need it.